### PR TITLE
Vipul

### PR DESCRIPTION
**Pull request by codervipul775**
I have added a pattern printing question of python. Please merge it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,9 +134,9 @@ Copy the next template into your file, delete the boilerplate data and fill the
 
 ```
 ---
-name: YOURNAME
-institution/company: INSTITUTION_NAME
-github: USERNAME
+name: Vipul
```
Comment (hasan-py) [inline README.md:137]: Why you changed on the mai readme file??

File: README.md [hunk 2]
```diff
@@ -151,7 +151,7 @@ Create/Upload your code in the folder following the convention `Add Code Here`
 - Below is `an example` to add file properly
 - You can follow up `any languages`
 ```
-Add Code Here/PYTHON/<YOUR-FILERNAME>.py
+Add Code Here/PYTHON/pattern.py
```
Comment (hasan-py) [inline README.md:154]: Why you changed here?? Its the main readme file

**Review by hasan-py — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (codervipul775) [reply README.md:154]: I don't know how to do a task . I was learning. So, it"s by mistake

**Comment by codervipul775** [comment]
Sir I have added a new pull request names as VIPUL_NEW. Please look at that once.

**Pull request closed without merging**